test/sample-sync-doc-definitions-spec.js

```diff
@@ -436,14 +437,14 @@ describe('The sample-sync-doc-definitions sync function', function() {
         oldDoc,
         expectedDocType,
         [
-          'documents of this type cannot be replaced',
           'cannot change "businessId" property',
```

Comment:
Because this error was generated by a custom validation function, it is intentionally not included in the validation-error-message-formatter module.